models/base_model.py

```diff
@@ -894,13 +896,20 @@ def save_networks_img(self, data):
             net = getattr(self, "net" + name)
             path = self.opt.checkpoints_dir + self.opt.name + "/networks/" + name
             if not "Decoder" in name:
-                temp = net(self.real_A)
+                y_0 = self.gt_image
+                y_cond = self.cond_image
+                mask = self.mask
+                noise = None
+
+                noise, noise_hat = net(y_0, y_cond, mask, noise)
+                # temp = net(self.real_A)
+                temp = noise_hat
             else:
                 temp = net(self.netG_A(self.real_A).detach())
             make_dot(temp, params=dict(net.named_parameters())).render(
                 path, format="png"
             )
-            paths.append(path)
+            # paths.append(path)
```

Comment:
Was this commented on purpose?

Reply:
Yes, toi avoid loading this image in visdom (image too large)